dot_env_example

```diff
@@ -7,9 +7,9 @@ DOCKERHUB_USERNAME="fjodorvr"
 VERSION="v3.0.1"
 ENV_FOR_DYNACONF=development
 PREFECT_SERVER_API_HOST='0.0.0.0'
-PREFECT_API_DATABASE_CONNECTION_URL="postgresql+asyncpg://username:password@db:5432/prefect"
+PREFECT_API_DATABASE_CONNECTION_URL="postgresql+asyncpg://postgres:postgres@db:5432/prefect"
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
 POSTGRES_DB=prefect
 POSTGRES_PORT=5432
-API_URL=http://0.0.0.0:4200/api
+API_URL=http://127.0.0.1:4200/api
```

Comment:
It is not the same, it might be that the application fails in production?

Reply:
Looking in the code I did find some hardcoded stuff: `prefect server start --host 0.0.0.0`.